### DISCUSSION TITLE
fwtm lite

### DISCUSSION THREAD

**Discussion by Pablovse** [FrSkyRC/ETHOS-Feedback-Community#2924] [Q&A] [answered]
i have a foot pedal that has a Taranis 9x with no Bluetooth, I need to transmit and receive that signal on my x20 pro, I'm waiting to get the FWTM LITE to connect in the back of the radio, what's the process to have them connect every time that I turn them on, the reason of this is because I fly with the pedals as throttle and rudder due to a disability

**Comment by bsongis-frsky** [comment, marked as answer]
It's available, but only in 1.5.0 nightlies for now!
